packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+                        {{ $t('mainTitle') }}
+                    </a>
+                </h2>
+
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ $t('textLine1') }}
+                </p>
+                <p :class="$style['c-cookieBanner-text']">
+                    {{ $t('textLine2') }}
+                </p>
+                <p
+                    :class="$style['c-cookieBanner-text']">
+                    <i18n
+                        path="textLine3">
+                        <template #cookiePolicy>
+                            <a
+                                :href="$t('cookiePolicyLinkUrl')"
+                                target="_blank">
+                                <span>{{ $t('cookiePolicyLinkText') }}</span>
+                            </a>
+                        </template>
+                    </i18n>
+                </p>
+            </div>
+
+            <div :class="$style['c-cookieBanner-CTA']">
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    is-full-width
+                    @click.native="acceptActions">
+                    {{ $t('acceptButtonText') }}
+                </button-component>
+                <button-component
+                    tabindex="0"
+                    role="button"
+                    button-type="ghost"
+                    is-full-width
+                    @click.native="nonAcceptActions">
+                    {{ $t('nonAcceptButtonText') }}
+                </button-component>
+            </div>
+        </div>
+        <legacy-banner
+            v-else
+            @hide-legacy-banner="hideBanner = true"
+        />
+    </div>
+</template>
+
+<script>
+import { globalisationServices } from '@justeat/f-services';
+import { VueGlobalisationMixin } from '@justeat/f-globalisation';
+
+import ButtonComponent from '@justeat/f-button';
+import '@justeat/f-button/dist/f-button.css';
+
+import LegacyBanner from './LegacyBanner.vue';
+
+import tenantConfigs from '../tenants';
+
+export default {
+    name: 'CookieBanner',
+
+    components: {
+        ButtonComponent,
+        LegacyBanner
+    },
+
+    mixins: [VueGlobalisationMixin],
+
+    props: {
+
+        locale: {
+            type: String,
+            default: 'en-GB'
+        },
+
+        isHidden: {
+            type: Boolean,
+            default: false
+        },
+
+        showLegacyBanner: {
+            type: Boolean,
+            default: false
+        },
+
+        cookieExpiry: {
+            type: Number,
+            default: 7776000
+        }
+    },
+
+    data () {
+        const locale = globalisationServices.getLocale(tenantConfigs, this.locale, this.$i18n);
+        const localeConfig = tenantConfigs[locale];
+        const theme = globalisationServices.getTheme(locale);
+
+        return {
+            config: { ...localeConfig },
+            tenantConfigs,
+            theme,
```

Comment:
You don't need these I believe, if you're using the `VueGlobalisationMixin`

Reply:
Hmmm - tried removing these and it all breaks. :(